.github/workflows/build-test.yml

```diff
@@ -25,18 +25,6 @@ env:
   IS_HOTFIX: ${{ contains(github.event.pull_request.title, 'hotfix') }}
 
 jobs:
-  check-is-hotfix:
-    name: Check if PR is hotfix
-    runs-on: ubuntu-latest
-    outputs:
-      is_hotfix: ${{ steps.check-is-hotfix.outputs.is_hotfix }}
-    env:  
-      IS_HOTFIX: ${{ contains(github.event.pull_request.title, 'hotfix') }}
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/actions/check-is-hotfix
-        id: check-is-hotfix
```

Comment:
GH actions doesn't support using env vars in `if` conditions, that's why we needed this job